### PR TITLE
OCPBUGS#840: Add a version tag

### PR DESCRIPTION
**Pull request by xenolinux**
Add a version tag to the must-gather image.

Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OCPBUGS-840

Link to docs preview: https://56122--docspreview.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data

QE review: @duanwei33 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by opayne1 — COMMENTED** [review]
/lgtm

File: modules/gathering-data-specific-features.adoc [hunk 1]
```diff
@@ -53,7 +53,7 @@ endif::openshift-dedicated[]
 |Data collection for OpenShift Shared Resource CSI Driver.
 
 ifndef::openshift-dedicated[]
-|`registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8`
+|`registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v<installed-version-LSO>`
```
Comment (bergerhoffer) [inline modules/gathering-data-specific-features.adoc:56]: Replaceable values should use underscores, not dashes. Can you please fix before we merge?

Comment (xenolinux) [reply modules/gathering-data-specific-features.adoc:56]: Addressed

**Pull request merged**